wp-content/themes/core/assets/pcss/admin.pcss

```diff
@@ -9,4 +9,6 @@
 @import "_utilities.pcss";
 
 /* Global Theme Editor Styles */
-@import "typography/editor.pcss";
+@import "global/reset.pcss";
+@import "typography/anchors.pcss";
```

Comment:
I reworked some selectors so that admin styles pick up the limited reset rules we have, and can use the same `anchors.pcss` partial as the theme stylesheet.